mscore/pianoroll/pianolevelsfilter.cpp

```diff
@@ -9,6 +9,24 @@
 
 namespace Ms {
 
+
+static const char* SCOPE = "PianoLevelsFilter";
+static const char* STRN_NOTE_ON_NAME = QT_TRANSLATE_NOOP(SCOPE, "Note on time");
```

Comment:
Sorry, once again I don't think this works: while the compiler (actually the preprocessor) certainly replaces  `SCOPE` here, 'lupdate` is just way to dumb for that.